Resources/Prototypes/Entities/Objects/Specific/Medical/morgue.yml

```diff
@@ -44,6 +44,10 @@
   - type: EntityStorage
     capacity: 1
     isCollidableWhenOpen: true
+    whitelist: # Use a tag whitelist rather than filtering for PerishableComponent to avoid chefs using body bags for food preservation
```

Comment:
freezers hold 30 items and can easily be mass produced so this has no meaning balance wise. it just goes against the immersive sim aspect

Reply:
That's true. I think I'm just kinda a hater about body bags. If downstreams don't want this lmk, I'll put in a PR to remove this row